firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -1149,17 +1166,12 @@ protocol TabWebViewDelegate: AnyObject {
     func tabWebViewShouldShowAccessoryView(_ tabWebView: TabWebView) -> Bool
 }
 
-class TabWebView: WKWebView, MenuHelperWebViewInterface, ThemeApplicable {
-    private struct UX {
-        static let documentLoadingViewAnimationDuration: CGFloat = 0.3
-    }
-
+class TabWebView: WKWebView, MenuHelperWebViewInterface, ThemeApplicable, FeatureFlaggable {
```

Comment:
We are removing code under `TabWebView`, but adding `FeatureFlaggable`? Can this be cleaned up?
```suggestion
class TabWebView: WKWebView, MenuHelperWebViewInterface, ThemeApplicable {
```